packages/common/src/HardhatConfig.ts

```diff
@@ -1,6 +1,8 @@
 import { HardhatConfig } from "hardhat/types";
 
 const { getNodeUrl, mnemonic } = require("./TruffleConfig");
+import { HRE } from "./hardhat/plugins/ExtendedWeb3";
```

Comment:
could you also comment on this `HRE` vs the previously commonly used `hre`? what is the distinction?

Reply:
HRE is a type that I created in the ExtendedWeb3 plugin to correctly reflect the components we add to the hre. So it's basically the vanilla web3 + some extra fields that we add in our setup, I will add a comment on this!

Reply:
ok awesome, thanks!